.github/workflows/lint.yml

```diff
@@ -30,9 +30,15 @@ jobs:
     - uses: actions/checkout@v3
     - name: check-patches.sh
       run: bash scripts/check-patches.sh
+  python:
+     strategy:
+       fail-fast: false
+       matrix:
+         version: ['3.11', '3.10', '3.9', '3.8']
```

Comment:
I'm ok to drop support for python3 < 3.10.

Reply:
I don't really think it hurts anything to keep the older ones if we are not writing code that necessitates a newer version but I do not feel strongly about it so I can drop it if you insist, we are pretty much the only two people who work in this repo anyways.